terraform_files/baremetal_infra_env/limit_ip_dhcp_range.xsl

```diff
@@ -12,11 +12,11 @@
   <xsl:template match="/network/ip[@family='ipv4']/dhcp/range">
```

Comment:
unless I'm missing something, I see no reason not to apply it to the none-platform template as well right?

Reply:
btw if the answer above is true we can have just 1 instance of that ``xsl`` file, place it under ``terraform_files`` and reference it like ``xslt = file("../limit_ip_dhcp_range.xsl")``

Reply:
Done